app/test_image.py

```diff
@@ -29,7 +32,7 @@ def main(src_path, bck_path, pretrained_model, output_path, output_type):
     model.eval()
 
     #load pretrained model
-    pretrained_state_dict = torch.load(pretrained_model)
+    pretrained_state_dict = torch.load(config["pretrained_model"],map_location='cuda:0')
```

Comment:
I think it is better to load the state_dict first to cpu and then load to cuda if available. The above code would fail on a device without any cuda capable devices.

`map_location='cpu'`

Reply:
Yes, that's right. Maybe should be configured by the user.